### PR TITLE
Crates are mailed with manifests taped to the outside

### PR DESCRIPTION
**Pull request by ghost**
Seriously this seems like a no-brainer feature when you make it so manifests give cargo techs and quarterjews extra points, locked crates having the manifests on the inside is absolutely stupid

shiet

### CODE REVIEW

**Comment by mrpain666** [comment]
Is it possible to reattach them once I rip them off? Can I even rip them off? Do I need to rip them off to stamp them?

**Comment by paprka** [comment]
Yes, you need to rip them off to stamp them, no, you can't reattach them.

**Comment by Chocobro** [comment]
do we still send it back inside the crate?
:+1:

**Comment by RandomMarine** [comment]
:+1: This makes cargo happy.

**Comment by paprka** [comment]
You never needed to send it back inside of crates, it doesn't check for that.

**Comment by RandomMarine** [comment]
Sometimes there are missing contents, pap

**Comment by paprka** [comment]
Well it's not like there was any way to tell without getting it opened anyway, meh.

**Comment by Chocobro** [comment]
alt+click is helpful for checking manifests. the possible errors are the contents of the manifest and the station destination. pet crates have empty manifests though, so it's a bit grey in those crates.

**Comment by JJRcop** [comment]
@Chocobro There's one more error you can get, the number of crates delivered!

**Comment by paprka** [comment]
Might want to bug report empty pet crate manifests.

Also, again, this doesn't affect much on secure crates. Having a manifest at all, even if you cant' check the contents, is better. A crate scanner would be excellent but I'm too lazy to implement it, this solves the main problem of not getting a manifest at all out of locked crates most of the time.

**Comment by Jalleo** [comment]
^^ it is reported.

**Comment by Chocobro** [comment]
@JJRcop i'm pretty sure if that error happens you don't get a manifest to stamp. since it's in the crate. which isn't there.
could be wrong on this.

also yeah, pet crate manifests being empty is reported already. https://github.com/tgstation/-tg-station/issues/6392

crate scanner would be super fresh. hmu if you need sprites.

**Comment by JJRcop** [comment]
@Chocobro It adds 1 or 2 to the reported number of crates arriving.
[/code/controllers/subsystem/shuttles.dm](https://github.com/tgstation/-tg-station/blob/f6380f385e2ca65b7cf14424a7d1dd488022ebe8/code/controllers/subsystem/shuttles.dm#L259)

**Comment by Jordie0608** [comment]
Changelog

**Comment by Chocobro** [comment]
@JJRcop neat! the more you know~*

**Comment by paprka** [comment]
changelog added

**Comment by Razharas** [comment]
Didnt cheri already merge this?
Im having deja vu

**Comment by Miauw62** [comment]
sadly, this PR has a) merge conflicts and b) duplicated code.
If anybody wants this in the game, they will have to manually redo this code.

**Comment by Cheridan** [comment]
Which part is duplicated?

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -270,6 +284,15 @@
 	return 1
 
 /obj/structure/closet/crate/attack_hand(mob/user as mob)
+	if(manifest)
```
Comment (Cheridan) [inline code/game/objects/structures/crates_lockers/crates.dm]: These bits you mean?

I'll handle it and make a manual merge PR.

**Comment by Cheridan** [comment]
soon :^)

**Comment by optimumtact** [comment]
Any day now @Cheridan

**Pull request closed without merging**